src/protocols/dns.cpp

```diff
@@ -194,8 +194,6 @@ namespace dns {
  * This is the internal object thanks to which Request is movable. Of
  * course, RequestImpl is not movable, since its address is passed to
  * one of the many evnds delayed requests functions.
- *
- * \remark You should not use this class directly, use Request instead.
```

Comment:
I think that specific line was misleading because RequestImpl was in fact directly used by the Resolver. On second thought, do we need to document RequestImpl? It's an internal class only defined inside dns.cpp.